app/views/curation_concerns/base/_form_progress.html.erb

```diff
@@ -43,6 +43,6 @@
     <%= link_to t(:'helpers.action.cancel'),
                 sufia.dashboard_index_path,
                 class: 'btn btn-default' %>
-    <%= f.submit 'Save', class: 'btn btn-primary', onclick: "confirmation_needed = false;", id: "with_files_submit", name: "save_with_files" %>
+    <%= f.submit class: 'btn btn-primary', onclick: "confirmation_needed = false;", id: "with_files_submit", name: "save_with_files" %>
```

Comment:
Should you be using `t('sufia.helpers.submit.create)` (or some such) here with the new `create` translation below or is that automagic?